class CollectError(Exception):
    pass

class ExperimentMismatchError(CollectError):
    pass

